app/controllers/admin/categories_controller.rb

```diff
@@ -48,5 +49,9 @@ def set_category
       def category_params
         params.require(:category).permit(:id, :name_en, :name_ja)
       end
+
+      def clear_cache
```

Comment:
__[Reek]__

[UtilityFunction] Admin::CategoriesController#clear_cache doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [Sider](https://sider.review) -->